state-chain/custom-rpc/src/lib.rs

```diff
@@ -507,8 +507,8 @@ pub trait CustomApi {
 		at: Option<state_chain_runtime::Hash>,
 	) -> RpcResult<Option<FailingWitnessValidators>>;
 
-	#[method(name = "decode_runtime_event")]
-	fn cf_decode_runtime_events(&self, events: Vec<String>) -> RpcResult<Vec<String>>;
+	#[method(name = "block_event")]
+	fn cf_get_events(&self, at: Option<state_chain_runtime::Hash>) -> RpcResult<Vec<String>>;
```

Comment:
Also, let's keep the method name and rpc method name consistent - cf_get_events and "get_events" is fine.